### PR TITLE
Minor changes for kani release

### PR DESCRIPTION
**Pull request by Firstyear**

### CODE REVIEW

**Review by yaleman — APPROVED** [review]

**Comment by Firstyear** [comment]
@micolous We're gonna need to do something to get away from the git rev versions here so we can do releases in future I think.

File: attestation-ca/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 [package]
 name = "webauthn-attestation-ca"
 version = "0.1.0"
+authors = ["William Brown <william@blackhats.net.au>"]
 edition = "2021"
+rust-version = "1.70.0"
+license = "MPL-2.0"
+description = "Webauthn Attestation Library"
+# documentation = "https://docs.rs/kanidm/latest/kanidm/"
+repository = "https://github.com/kanidm/webauthn-rs"
```
Comment (yaleman) [inline attestation-ca/Cargo.toml:10]: These could inherit from the workspace

**Review by yaleman — APPROVED** [review]

**Comment by Firstyear** [comment]
I'd like @micolous input before I proceed on this :)

**Review by micolous — APPROVED** [review]

**Review by micolous — CHANGES_REQUESTED** [review]
Actually, this breaks on CI

File: fido-key-manager/src/main.rs [hunk 2]
```diff
@@ -13,7 +13,7 @@ use std::time::Duration;
 use tokio_stream::StreamExt;
 #[cfg(feature = "solokey")]
 use webauthn_authenticator_rs::ctap2::SoloKeyAuthenticator;
-use webauthn_authenticator_rs::prelude::WebauthnCError;
+// use webauthn_authenticator_rs::prelude::WebauthnCError;
```
Comment (micolous) [inline fido-key-manager/src/main.rs:16]: This should be gated on `#[cfg(feature = "solokey"]` to fix the warning.

**Comment by Firstyear** [comment]
@micolous We had a bunch of issues generally, because we can't use git versions of things on crates.io, which meant btleplug was problematic. So we probably generally have a bunch of cleanup to get to a releasable state, which I want to do "sooner than later"

File: fido-key-manager/src/main.rs (hunk 2 above)
Comment (Firstyear) [reply fido-key-manager/src/main.rs:16]: K

**Comment by Firstyear** [comment]
I dunno what I just did, but I think I cooked this PR 🤣

I think I close it in favour of another attempt

**Pull request closed without merging**